### ISSUE TITLE
true braille pixels in images

### ISSUE THREAD

**Issue by benjajaja** [nidhoggfgg/rsille#1]
When drawing an image on canvas, it only ever generates full `⣿` characters with a foreground color.

What would it take to generate different braille characters, similar to when drawing a line? A character could also have a background color, that is, for each character it could be possible to render 2x4 pixels with a two-color palette.

**Comment by nidhoggfgg** [comment]
yeah, that's right. use background color and not just full `⣿` can make it much better. i would like to implement it. but unfortunately, recently i should remake the canvas and the animation firstly, then a full 3D render (not like that simple in extra/object3d.rs now). when i done these things, i will do what you said.
